### PR TITLE
feat: Update to Angular 15

### PR DESCRIPTION
**Pull request by JeanMeche**
This commit add the support for Angular v15 (with legacy components).

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
Hi all, started looking things over since the angular-themes alpha is in here. It might be better to develop against a running dev branch along side this feature branch or you can use this deployed version if it makes it easier.
https://blui-angular-showcase.web.app/templates/dashboard

this is just a quick overview of what I see, so I would check against everything because there are more that I didn't capture.

Some items I've noticed
* focus & hover state colors on chips are slightly off on all variants, along with a couple of other items you'll see in the screen shots.

image
image

* snackbar

image

* button toggles

image

* disabled accordion

image

**Comment by JeffGreiner-eaton** [comment]
just adding this warning I viewed in the terminal while starting the showcase..

image

File: src/styles.scss [hunk 1]
```diff
@@ -95,13 +93,13 @@ $warn: (
...
-    $blui-blue-dark-warn: theme-bases.$blui-blue-dark-warn;
+// .blui-blue-dark {
+//     $blui-blue-dark-primary: theme-bases.$blui-blue-dark-primary;
+//     $blui-blue-dark-accent: theme-bases.$blui-blue-dark-accent;
+//     $blui-blue-dark-warn: theme-bases.$blui-blue-dark-warn;
 
-    @include theme-bases.dark-theme($blui-blue-dark-primary, $blui-blue-dark-accent, $blui-blue-dark-warn);
-}
+//     @include theme-bases.dark-theme($blui-blue-dark-primary, $blui-blue-dark-accent, $blui-blue-dark-warn);
+// }
```
Comment (daileytj) [inline src/styles.scss:102]: Is this something we can just remove?

**Comment by daileytj** [comment]
@JeffGreiner-eaton @JeanMeche The style issues are all going to need to be fixed in the themes or component repo. I'd recommend getting this merged and then addressing these issues with follow up stories in the appropriate repos.

**Pull request merged**